lib/kb_Msuite/Utils/CheckMUtil.py

```diff
@@ -131,12 +131,12 @@ def build_checkM_lineage_wf_plots(self, bin_folder, out_folder, plots_folder,
                                       all_seq_fasta_file, tetra_file):
 
         # first build generic plot for entire dataset
-        log('Creating basic QA plot (checkm bin_qa_plot) ...')
-        bin_qa_plot_options = {'bin_folder': bin_folder,
-                               'out_folder': out_folder,
-                               'plots_folder': plots_folder
-                               }
-        self.run_checkM('bin_qa_plot', bin_qa_plot_options, dropOutput=True)
+#         log('Creating basic QA plot (checkm bin_qa_plot) ...')
```

Comment:
This option has been removed in the newest checkM release. Commented out for now, removed properly in a PR to come.